website/docs/r/data_protection_backup_policy_postgresql.html.markdown

```diff
@@ -0,0 +1,154 @@
+    }
+  }
+
+  retention_rules {
+    name             = "thursday"
+    duration         = "P1W"
+    tagging_priority = 25
+    tagging_criteria {
+      days_of_the_week = ["Thursday"]
+      schedule_times   = ["2021-05-23T02:30:00Z"]
+    }
+  }
+
+  retention_rules {
+    name             = "monthly"
+    duration         = "P1D"
+    tagging_priority = 30
+    tagging_criteria {
+      weeks_of_the_month = ["First", "Last"]
+      days_of_the_week   = ["Tuesday"]
+      schedule_times     = ["2021-05-23T02:30:00Z"]
+    }
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Backup Policy Postgre Sql. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Backup Policy Postgre Sql should exist. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+* `vault_name` - (Required) The name of the Backup Vault where the Backup Policy Postgre Sql should exist. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+* `backup_rules` - (Required) A `backup_rules` block as defined below. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+* `default_retention_duration` - (Required) The duration of default retention rule. It should follow `ISO 8601` duration format. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+---
+
+* `retention_rules` - (Optional) One or more `retention_rules` blocks as defined below. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+---
+
+A `backup_rules` block supports the following:
+
+* `name` - (Required) The name which should be used for this backup rule. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+* `repeating_time_intervals` - (Required) Specifies a list of repeating time interval. It should follow `ISO 8601` repeating time interval . Changing this forces a new Backup Policy Postgre Sql to be created.
+
+---
+
+A `retention_rules` block supports the following:
+
+* `name` - (Required) The name which should be used for this retention rule. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+* `duration` - (Required) Duration of deletion after given timespan. It should follow `ISO 8601` duration format. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+* `tagging_criteria` - (Required) A `tagging_criteria` block as defined below. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+* `tagging_priority` - (Required) Retention Tag priority. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+---
+
+A `tagging_criteria` block supports the following:
+
+* `absolute_criteria` - (Optional) Possible values are `AllBackup`, `FirstOfDay`, `FirstOfWeek`, `FirstOfMonth` and `FirstOfYear`. Changing this forces a new Backup Policy Postgre Sql to be created.
```

Comment:
what does `absolute_criteria` mean in this context here? could we detail this?

Reply:
It's a kind of tagging criteria (when to run a backup). For example, `FirstOfDay` means it will backup everyday and use the first successful backup of the day. I'll add explanations to these values.

Reply:
It's a kind of tagging criteria (which backup should be retained). For example, FirstOfDay means retain the first successful backup of the day. I'll add explanations to these values.